torchinductor/codegen/triton.py

```diff
@@ -764,6 +764,19 @@ def __init__(self, scheduler):
     def group_fn(self, sizes):
         return tuple(V.graph.sizevars.simplify(sympy_product(s)) for s in sizes)
 
+    def group_fn_NHW_C(self, sizes):
```

Comment:
I think a better way to handle this would be to reorder the loops in simplify_and_reorder.

Reply:
This is to specify the group of `TritonTemplateKernel` like conv, as a reference for the fusable pointwise `TritonKernel` to use `simplify_reorder_and_tile()` later.